pkg/agent/controller/serviceexternalip/controller.go

```diff
@@ -111,7 +111,7 @@ func NewServiceExternalIPController(
 		assignedIPs:           make(map[string]sets.Set[string]),
 		linkMonitor:           linkMonitor,
 	}
-	ipAssigner, err := ipassigner.NewIPAssigner(nodeTransportInterface, "", linkMonitor)
+	ipAssigner, err := ipassigner.NewIPAssigner(nodeTransportInterface, "", linkMonitor, false)
```

Comment:
@xliuxu I set it as false to keep the behavior the same as before for ServiceExternalIP. We may consider to add a new option for ServiceExternalIP like Egress?

Reply:
The `ServiceExternalIP` feature does not create new interfaces